spec/features/delius_import_job_feature_spec.rb

```diff
@@ -49,23 +49,52 @@
 
     ENV['DELIUS_EMAIL_FOLDER'] = 'delius_import_feature'
     ENV['DELIUS_XLSX_PASSWORD'] = 'secret'
+  end
+
+  context "when the team is associated with an LDU" do
+    before do
+      create(:team, code: 'A')
+    end
+
+    it "imports the Delius spreadsheet and creates case information" do
+      visit prison_summary_pending_path(prison)
+      expect(page).to have_content("Add missing information")
+      expect(page).to have_content(offender_no)
 
-    create(:team, code: 'A')
+      DeliusImportJob.perform_now
+
+      reload_page
+      expect(page).to have_content("Add missing information")
```

Comment:
I think this is a good assertion to use here - it's general enough that it shouldn't be too brittle - but I'm wondering how to make it more obvious to a reader what we're checking. 

You need prior knowledge of other bits of the app to "get" this test, not sure how much of a problem that is. 

I also have no concrete suggestions for improving it :shrug: so not sure how useful this is!

Reply:
On further reflection, maybe it could be helped with more structure, using context/it blocks to describe what the different stages/assertions are?

Reply:
I think our feelings about the structure of this file might change once we've got more contexts in – coming shortly in [POM-686](https://dsdmoj.atlassian.net/browse/POM-686) is functionality to map shadow teams to active teams. It might be worth having this discussion again at that point.